src/renderer/actions/FileCheckActions.ts

```diff
@@ -0,0 +1,79 @@
+import { Action, ActionCreator } from 'redux';
+
+/* Action types */
+
+// Stepper
+export const STEP_PROGRESS = 'STEP_PROGRESS';
+export const STEP_RESET = 'STEP_RESET';
+
+// Files
+export const CLEAR_FILES = 'CLEAR_FILES';
+export const SET_FILES = 'SET_FILES';
+
+
+/* Typescript interfaces */
+
+// Stepper
+export interface StepProgressAction extends Action {
+    type: 'STEP_PROGRESS';
+}
+
+export interface StepResetAction extends Action {
+    type: 'STEP_RESET';
+}
+
+// Files
+export interface FileData {
+    path: string;
+    size: string;
+}
+
+export interface SetFilesAction extends Action {
+    type: 'SET_FILES';
+    files: Array<FileData>;
+}
+
+export interface ClearFilesAction extends Action {
+    type: 'CLEAR_FILES';
+}
+
+
+/* Action functions*/
+
+// Stepper
+
+/**
+ * Progress one step. Increases the counter by 1
```

Comment:
Not a big fan of comments like these, the title of the function should imply what the function does. If you rename the function to 'incrementStep' or 'progressStepByOne', then this comment isn't necessary anymore :)